content/chapters/compute/lecture/slides.mdpp

```diff
@@ -172,7 +169,7 @@ struct task_struct {
 
 ### `fork`
 
-* Spawns child process
+* System call that spawns child process
```

Comment:
This is wrong. `fork()` isn't a syscall. It's a libc function that calls `clone()` (the actual syscall). I used that wording on purpose.